src/config.rs

```diff
@@ -106,7 +106,7 @@ pub struct Config {
     pub minus_file: Option<PathBuf>,
     pub minus_non_emph_style: Style,
     pub minus_style: Style,
-    pub navigate_regexp: Option<String>,
+    pub navigate_regexp: String,
```

Comment:
And if you wouldn't mind making it consistent in the internal variables that would be great too :)

Reply:
Sure, I changed all the navigate regexp to remove the "p".